atlp-rwanda/ecommerce-app-champions-bn
# #15  a seller should be able to mark a product as available or unavailable

#### What does this PR do?

- list available products in a seller collection
- make a product available
- make a product unavailable
- document endpoint with swagger
- add translation
#### Description 
based on expiry date 
 a seller I should be able to mark a product as available or unavailable manually in order to make it available for the buyers 
#### How should this be manually tested?
- clone the repo `git clone https://github.com/atlp-rwanda/ecommerce-app-champions-bn.git`
- checkout to `ft-makeProduct-available-unavailable `
- install the packages by `npm install`
- start the server `npm run dev`
- open your browser and type ` http://localhost:{PORT}/docs`
-  first login as a vendor i.e you must be having login credentials for the vendor and validated by two-factor-auth
- if don't have an account signup as a vendor
- create product `http://localhost:{PORT}/api/product/create`
- list products you have in your database
- once you get a list of products you can use productId in searchParam to make the product. available or unavailable
Note: make sure you have Cloudinary for photo storage
### Trello cardNumber#15
